wagtail/admin/ui/tables.py

```diff
@@ -197,12 +198,37 @@ def get_cell_context_data(self, instance, parent_context):
         return context
 
 
+class LiveStatusTagColumn(StatusTagColumn):
+    """Represents a live/draft status tag"""
+
+    def __init__(self, **kwargs):
+        super().__init__(
+            "status_string",
+            label=_("Status"),
+            sort_key="live",
+            primary=lambda instance: instance.live,
+            **kwargs,
+        )
+
+
 class DateColumn(Column):
     """Outputs a date in human-readable format"""
 
     cell_template_name = "wagtailadmin/tables/date_cell.html"
 
 
+class UpdatedAtColumn(DateColumn):
+    """Outputs the _updated_at date annotation in human-readable format"""
+
+    def __init__(self, **kwargs):
+        super().__init__(
+            "_updated_at",
```

Comment:
In case you aren't aware, the `_updated_at` is an annotation done on the view-level that's applied dynamically (i.e. it's not a real field, hence the leading underscore). It uses the latest log entry by default, but uses the latest revision's `created_at` if `RevisionMixin` is applied to the model.

This class is created so users don't have to know about this specific implementation detail.